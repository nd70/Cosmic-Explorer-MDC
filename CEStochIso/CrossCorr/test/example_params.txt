%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% example parameter file
%%%%%%%%%%%% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

% stochastic output files
writeNaiveSigmasToFiles true
writeSpectraToFiles true
writeStatsToFiles true
writeSensIntsToFiles true
writeCoherenceToFiles false
writeCalPSD1sToFiles false
writeCalPSD2sToFiles false

% high-pass parameters
useCascadeFilter1 true
useCascadeFilter2 true
doHighPass1 true
doHighPass2 true
highPassFreq1 11
highPassFreq2 11
highPassOrder1 16
highPassOrder2 16

% overalpping segments
doOverlap true

% do not use folded data
doSidereal false

% do not analyze less than 200s of data
minDataLoadLength 200 

% do not exclude any additional times
doBadGPSTimes false

% perform a time shift
doShift1 true
doShift2 false
ShiftTime1 1
ShiftTime2 0

% use Hanford-Livingston pair
ifo1 H1
ifo2 L1

% define properties of STAMP pixels
segmentDuration 60
numSegmentsPerInterval 3
ignoreMidSegment true

% duration of Hann window
hannDuration1 60
hannDuration2 60

% define frequency range and resolution
flow 10
fhigh 1800
deltaF 0.25

% define standard stochastic parameters
alphaExp 0
fRef 100
simOmegaRef 0
heterodyned false
resampleRate1 4096
resampleRate2 4096
bufferSecs1 2
bufferSecs2 2
nResample1 10
nResample2 10
betaParam1 5
betaParam2 5

% strain channel names
ASQchannel1 DCS-CALIB_STRAIN_C02
ASQchannel2 DCS-CALIB_STRAIN_C02

% frame types
frameType1 H1_HOFT_C02
frameType2 L1_HOFT_C02

% use default calibration
alphaBetaFile1 none
alphaBetaFile2 none
calCavGainFile1 none
calCavGainFile2 none
calResponseFile1 none
calResponseFile2 none

% never mask frequency bins at the pre-processing stage
doFreqMask false
freqsToRemove
nBinsToRemove

% paths to cachefiles
gpsTimesPath1 ./cachefiles/
gpsTimesPath2 ./cachefiles/
frameCachePath1 ./cachefiles/
frameCachePath2 ./cachefiles/

% do not use Monte Carlo
doDetectorNoiseSim false

% output file location
outputFilePrefix ./example
