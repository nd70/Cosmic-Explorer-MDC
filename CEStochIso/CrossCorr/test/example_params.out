orderfields(params)

ans = 

                                     ASQchannel1: 'LSC-STRAIN'
                                     ASQchannel2: 'LSC-STRAIN'
                                  ConstTimeShift: 0
                                          HfFile: ''
                    HfFileInterpolateLogarithmic: 1
                                      ShiftTime1: 0
                                      ShiftTime2: 0
                  SkyPatternDeclinationNumPoints: 0
                                  SkyPatternFile: ''
               SkyPatternRightAscensionNumPoints: 0
                                  SpHFreqIntFlag: 1
                                         SpHLmax: 0
                            TimingTransientFile1: ''
                            TimingTransientFile2: ''
                             UnphysicalTimeShift: 0
                                  alphaBetaFile1: 'none'
                                  alphaBetaFile2: 'none'
                                        alphaExp: 0
                                        azimuth1: NaN
                                        azimuth2: NaN
                                 badGPSTimesFile: ''
                                      betaParam1: 5
                                      betaParam2: 5
                                     bufferSecs1: 2
                                     bufferSecs2: 2
                                        cacheMat: '/archive/home/ethrane/matapps/PreProcessing/cache.mat'
                                 calCavGainFile1: 'none'
                                 calCavGainFile2: 'none'
                                calResponseFile1: 'none'
                                calResponseFile2: 'none'
                                  ddmmyyyyhhmmss: '17-Sep-2013 02:03:55'
                                          deltaF: 0.2500
                              doAllSkyComparison: 0
                                   doBadGPSTimes: 0
                                       doCombine: 0
                                doConstTimeShift: 0
                              doDetectorNoiseSim: 0
                                   doDirectional: 0
                                      doFreqMask: 0
                                     doHighPass1: 1
                                     doHighPass2: 1
                                  doInjFromFile1: 0
                                  doInjFromFile2: 0
                                      doMCoffset: 0
                                    doMonteCarlo: 0
                          doNarrowbandRadiometer: 0
                                       doOverlap: 1
                                        doShift1: 0
                                        doShift2: 0
                                      doSidereal: 0
                        doSimulatedDetectorNoise: 0
                          doSimulatedPointSource: 0
                               doSimulatedSkyMap: 0
                            doSphericalHarmonics: 0
                   doTimingTransientSubtraction1: 0
                   doTimingTransientSubtraction2: 0
                                            fRef: 100
                                          fbase1: NaN
                                          fbase2: NaN
                                           fhigh: 1800
                                            flow: 10
                                 frameCachePath1: './cachefiles/'
                                 frameCachePath2: './cachefiles/'
                                  frameDuration1: -1
                                  frameDuration2: -1
                                      frameType1: 'H1_RDS_C03_L2'
                                      frameType2: 'L1_RDS_C03_L2'
                                   freqsToRemove: []
                              gammaLM_coeffsPath: ''
                                   gpsTimesPath1: './cachefiles/'
                                   gpsTimesPath2: './cachefiles/'
                                   hannDuration1: 60
                                   hannDuration2: 60
                                     heterodyned: 0
                                   highPassFreq1: 8
                                   highPassFreq2: 8
                                  highPassOrder1: 6
                                  highPassOrder2: 6
                                            ifo1: 'H1'
                                            ifo2: 'L1'
                                ignoreMidSegment: 1
                                     injChannel1: ''
                                     injChannel2: ''
                              injFrameCachePath1: ''
                              injFrameCachePath2: ''
                               injFrameDuration1: -1
                               injFrameDuration2: -1
                                injGPSTimesPath1: '<auto>'
                                injGPSTimesPath2: '<auto>'
                                      injPrefix1: 'H1'
                                      injPrefix2: 'L1'
                                       injScale1: 1
                                       injScale2: 1
                               intFrameCachePath: './intermediate/'
                                    intermediate: 0
                                     jobDuration: 315
                                       jobNumber: 2
                                        jobsFile: 'example_jobs.txt'
                            jobsFileCommentStyle: 'matlab'
                         maxCorrelationTimeShift: 0
                                    maxDSigRatio: 1000
                                        maxMCoff: 0.6000
                           maxSegmentsPerMatfile: 60
                                    minDSigRatio: 0
                               minDataLoadLength: 200
                                        minMCoff: 0.1000
                                   nBinsToRemove: []
                                      nResample1: 10
                                      nResample2: 10
                                        numFreqs: 7161
                          numSegmentsPerInterval: 3
                                       numTrials: 1
                                outputFilePrefix: './example'
                                      paramsFile: 'example_params.txt'
                                   resampleRate1: 4096
                                   resampleRate2: 4096
                                    saveGPS_diff: 0
                                 segmentDuration: 60
                                         segsMat: '/archive/home/ethrane/matapps/PreProcessing/segs.mat'
                                      signalType: 'const'
                                    simOmegaRef1: 0
                                    simOmegaRef2: 0
    simulatedDetectorNoiseInterpolateLogarithmic: 1
            simulatedDetectorNoiseMapBufferDepth: 0
       simulatedDetectorNoiseMapHalfRefillLength: 0
                simulatedDetectorNoiseNoRealData: 0
                 simulatedDetectorNoisePowerSpec: ''
                simulatedPointSourcesBufferDepth: 0
                       simulatedPointSourcesFile: ''
           simulatedPointSourcesHalfRefillLength: 0
     simulatedPointSourcesInterpolateLogarithmic: 1
             simulatedPointSourcesMakeIncoherent: 0
                 simulatedPointSourcesNoRealData: 0
                  simulatedPointSourcesPowerSpec: ''
                      simulatedSkyMapBufferDepth: 0
                       simulatedSkyMapConvertDeg: 10
                      simulatedSkyMapConvertLmax: 0
                             simulatedSkyMapFile: ''
                      simulatedSkyMapFileNumBins: 1
                         simulatedSkyMapFileType: 1
                 simulatedSkyMapHalfRefillLength: 0
                      simulatedSkyMapInjectAsSpH: 1
                 simulatedSkyMapInjectTimeDomain: 1
           simulatedSkyMapInterpolateLogarithmic: 1
                   simulatedSkyMapMakeIncoherent: 0
                       simulatedSkyMapNoRealData: 0
                        simulatedSkyMapPowerSpec: ''
                                  simulationPath: ''
                                           site1: 'LHO'
                                           site2: 'LLO'
                                       startTime: 816070181
                              suppressFrWarnings: 0
                     useSignalSpectrumHfFromFile: 0
                               useSkyPatternFile: 0
                                         verbose: 0
                            writeCalPSD1sToFiles: 0
                            writeCalPSD2sToFiles: 0
                                writeCohFToFiles: 0
                           writeCoherenceToFiles: 0
                         writeNaiveSigmasToFiles: 1
                      writeOptimalFiltersToFiles: 0
            writeOverlapReductionFunctionToFiles: 0
                            writeResultsToScreen: 1
                            writeSensIntsToFiles: 1
                             writeSpectraToFiles: 1
                               writeStatsToFiles: 1

diary
